tests/purs/failing/ExtraRecordField.out

```diff
@@ -4,6 +4,8 @@ at tests/purs/failing/ExtraRecordField.purs:9:13 - 9:54 (line 9, column 13 - lin
 
   Type of expression contains additional label [33mage[0m.
 
+while matching label [33mlast[0m
+while matching label [33mfirst[0m
```

Comment:
This is a very bad hint; the labels `last` and `first` are correct! The issue, as the message otherwise states, is with the label `age`.